rplidar.go

```diff
@@ -210,6 +216,10 @@ func (rp *Rplidar) Stream(ctx context.Context, errHandlers ...gostream.ErrorHand
 
```

Comment:
Do we want a resource is closed error for these functions or is unimplemented sufficient?

Reply:
unimplemented is sufficient